core/src/main/java/org/jruby/RubyEnumerable.java

```diff
@@ -32,22 +32,10 @@
 import org.jruby.common.IRubyWarnings.ID;
 import org.jruby.exceptions.JumpException;
 import org.jruby.exceptions.RaiseException;
-import org.jruby.runtime.Arity;
-import org.jruby.runtime.Block;
-import org.jruby.runtime.BlockBody;
-import org.jruby.runtime.BlockCallback;
-import org.jruby.runtime.CallBlock;
-import org.jruby.runtime.CallBlock19;
-import org.jruby.runtime.Helpers;
-import org.jruby.runtime.JavaInternalBlockBody;
-import org.jruby.runtime.JavaSites;
+import org.jruby.runtime.*;
```

Comment:
:scissors: I hate when IDEA does that ... already tried forcing it not to do so twice :disappointed: